shared/jobs.lua

```diff
@@ -1,6 +1,9 @@
 QBShared = QBShared or {}
 QBShared.ForceJobDefaultDutyAtLogin = true -- true: Force duty state to jobdefaultDuty | false: set duty state from database last saved
-QBShared.Jobs = {
+QBShared.QBJobsStatus = false -- true: qb-jobs resource is installed | false: qb-jobs resource is not installed
```

Comment:
```suggestion
QBShared.QBJobsStatus = GetResourceState("qb-jobs") ~= "missing" -- Dont touch
```

Reply:
Leave that flag as is. We don't want to automate that.

The reason is people have put a lot of work into theor cities.

It takes time to integrate everything.

They can set it to false and install the resource and still use it for
multi-job capabilities
Easy job creation for new jobs
The use of the /duty command

When they finish with the config filea for the exoatinf jobs they can switch to true to fully integrate qb-jobs to the system.

Which will reduce overall resource load and give all the other jobs the same features and capabilities that qb-jobs offers.

Reply:
-- true: qb-jobs resource is installed | false: qb-jobs resource is not installed